lms/static/js/spec/views/fields_helpers.js

```diff
@@ -63,6 +63,10 @@ define(['backbone', 'jquery', 'underscore', 'common/js/spec_helpers/ajax_helpers
             expect(view.$('.u-field-title').text().trim()).toContain(expectedTitle);
         };
 
+        var expectDropdownSrTitleToContain = function(view, expectedTitle) {
```

Comment:
It also would make the Jasmine tests themselves be more idiomatic, so you don't have to know what the helper function itself is asserting.